https://github.com/doxygen/doxygen/pull/8862
https://github.com/doxygen/doxygen/issues/8855

[sam: deleted msc hunk b/c we had it in a patch of our
own.]

From 5c3a4ad2773fc911b341b36b00528603afbe1602 Mon Sep 17 00:00:00 2001
From: albert-github <albert.tests@gmail.com>
Date: Wed, 27 Oct 2021 16:41:28 +0200
Subject: [PATCH] issue #8855 Unable to build 1.9.2

All other libraries have been declared STATIC so better to declare the cmls and mscgen libraries also STATIC.
---
 libxml/CMakeLists.txt    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libxml/CMakeLists.txt b/libxml/CMakeLists.txt
index 96c56534f9..596ed90887 100644
--- a/libxml/CMakeLists.txt
+++ b/libxml/CMakeLists.txt
@@ -11,7 +11,7 @@ set_source_files_properties(${GENERATED_SRC}/xml.l.h PROPERTIES GENERATED 1)
 
 FLEX_TARGET(xml xml.l ${GENERATED_SRC}/xml.cpp COMPILE_FLAGS "${LEX_FLAGS}")
 
-add_library(xml
+add_library(xml STATIC
 ${GENERATED_SRC}/xml.cpp
 ${GENERATED_SRC}/xml.l.h
 )
